src/BullaFrendLendV2.sol

```diff
@@ -469,6 +473,47 @@ contract BullaFrendLendV2 is BullaClaimControllerBase, ERC165, IBullaFrendLendV2
         emit TokenRemovedFromFeesWhitelist(token);
     }
 
+    /**
+     * @notice Allows admin to add a callback contract and selector to the whitelist
+     * @param callbackContract The contract address to whitelist
+     * @param selector The function selector to whitelist for this contract
+     */
+    function addToCallbackWhitelist(address callbackContract, bytes4 selector) external {
+        if (msg.sender != admin) revert NotAdmin();
```

Comment:
Might as well make this into an `onlyAdmin` modifier because there are several instances of this being used and readability helps